test/TestFunctions/Function1.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.IO;
```

Comment:
Most of the changes in this PR are restructuring the "direct load" test to move away from dynamic compilation and instead to use an actual test class library